delivery_roulier/views/carrier_account.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_roulier_carrier_account_tree" model="ir.ui.view">
+        <field name="model">carrier.account</field>
+        <field name="inherit_id" ref="base_delivery_carrier_label.carrier_account_view_tree"/>
+        <field name="arch" type="xml">
+            <field name="file_format" position="attributes">
+                <attribute name="invisible">1</attribute>
```

Comment:
why this?

Reply:
It is hard to have a generic selection of file_format. Each carrier has its own list, with its own labels, etc...
That is why, for each carrier, usually you should implement a `carrier_type_file_format` field to set your selection list and display it in the view.
But in this case, you don't want to see the `file_format` field, which is useless.

Of course, if we use the file_format as fallback,it becomes an issue, but once again, can we really use a standard list of multiple carriers?